src/app/server/views/app.hbs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE HTML>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>{{config.title}}</title>
+        <link rel="stylesheet" href="/css/{{config.appType}}App.css">
+    </head>
+    <body>
+        <div id="app">
+        </div>
```

Comment:
I think I expected separate template for the separate app types, like `basic.hbs` with 
```
<div id="app">
<basic-app :title="{{ title }}"></basic-app>
</div>
```

Partly to allow the passing of props to the root component, partly for easier testing (e.g. can test that a basic app config results in the correct template/component/css being rendered)